common/culture/cultures/ogre.txt

```diff
@@ -2,15 +2,12 @@
 	graphical_cultures = {
```

Comment:
Reminder: add hairs and beards.

Reply:
> 
> 
> Reminder: add hairs and beards.

On it!

Reply:
Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/188.